### ISSUE TITLE
Instruction.getDisassembly SystemError

### ISSUE THREAD

**Issue by 0vercl0k** [JonathanSalwan/Triton#657]
Hey there,

The below code triggers a `SystemError` exception (which I'm pretty sure it shouldn't):

```text
(C:\ProgramData\Anaconda2) c:\Users\over\Downloads\triton.x64.release.a9d4b9a8>python oops_getDisassembly.py
0x0:  1
Traceback (most recent call last):
  File "oops_getDisassembly.py", line 13, in <module>
    print disass(i)
  File "oops_getDisassembly.py", line 5, in disass
    i.getAddress(), i.getDisassembly()
SystemError: ..\Objects\stringobject.c:3900: bad argument to internal function
```

```python
from triton import *

def disass(i):
    return '0x%.16x: %s' % (
        i.getAddress(), i.getDisassembly()
    )

ctx = TritonContext()
ctx.setArchitecture(ARCH.X86_64)
 
i = Instruction('\xc3')
print i, i.getSize()
print disass(i)
```

I have reproduced this on Windows with the x64 artifacts available here: https://ci.appveyor.com/project/JonathanSalwan/triton/build/0.6.2600.

Cheers

**Comment by JonathanSalwan** [comment]
Thanks Axel. Fixed in the `dev-v0.6` with d65610a7f271803086b1fd7ade392d2a4e1a649f.

**Comment by 0vercl0k** [comment]
Awesome, thanks man!

2018-02-13 8:47 GMT-08:00 Jonathan Salwan <notifications@github.com>:
